src/List.mo

```diff
@@ -25,10 +51,26 @@ module {
   };
 
   /// Construct a list by pre-pending a value.
```

Comment:
```suggestion
  /// Add `x` to the head of `list`.
```
I think some beginners will think that you are reconstructing the entire list if you say "construct"